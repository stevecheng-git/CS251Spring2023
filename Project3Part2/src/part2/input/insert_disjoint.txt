idtgbs
fwjgup
turcan
oyvtcb
zqveat
spddhz
yienea
shuigt
lhmqxh
xlcwmp
ifespe
usozag
