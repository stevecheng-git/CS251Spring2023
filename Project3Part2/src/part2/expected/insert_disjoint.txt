0 0 0 0 0 0 19 2
0 8 64 0 4 0 83 10
0 -120 64 16 20 8 83 26
4 -56 64 -112 -108 8 83 90
4 -56 65 -112 -108 -120 91 90
68 -56 65 -110 -44 -56 123 90
68 -56 65 -14 -42 -56 123 90
84 -40 65 -10 -42 -55 127 90
84 -8 65 -10 -2 -55 -1 -38
84 -4 65 -10 -2 -21 -1 -38
124 -2 65 -10 -2 -21 -1 -38
125 -2 -63 -2 -2 -21 -1 -37
